IO-phone/package-lock.json

```diff
@@ -1,3 +1,5 @@
+<<<<<<< HEAD
+=======
```

Comment:
To są linie dodane przez git sygnalizujące merge conflict. Usuń zmiany w tym pliku